doc/source/news.rst

```diff
@@ -5,6 +5,50 @@
 News
 ====
 
+.. _release-11-0-6:
+
+Release 11.0.6 - 2021-08-29
+---------------------------
+
+Improvements
+------------
+
+* Added support for recovering on crash. (experimental)
+
+  * This is a experimental feature. Currently, this feature is still not stable.
+
+  * If Groonga crashes, it recovers the database automatically when it open a database for the first time since the crash. However, This feature can't recover the database automatically in all crash cases.
+    We need to recover the database manually depending on timing even if this feature enable.
+
+* [:doc:`reference/executables/grndb`] Added ``dump-wal`` command.
+
+* Added a new tool ``dump-wal.rb`` for dumping WAL.
+
+* 
+
+Fixes
+-----
+
+* Fixed a bug that Groonga doesn't clear lock when some threads open the same object around the same time.
+
+  If some threads open the same object around the same time, threads except for a thread that executes the opening object at first are waiting for opening the target object.
+  At this time, threads that are waited for an opening object take locks, but these locks are not released.
+  Therefore, these locks remain until Groonga's process is restarted in the above case.
+
+Known Issues
+------------
+
+* Currently, Groonga has a bug that there is possible that data is corrupt when we execute many additions, delete, and update data to vector column.
```

Comment:
What is **that** pointed to ?

Groonga has a bug that there is possible _that_ data ...

Maybe **that** is needless here?

---
Here is my translation from Japanese sentence https://github.com/groonga/groonga/pull/1255/files#diff-9222b1bb1ab202b29a630c4757a7214b0663fd403af91b16aa5f1c720df9f652R104-R105 :

Currently, Groonga has a bug that there is a possibility that the data is corrupt when us lots of added, deleted, and updated to a vector column.

Reply:
Thanks!
However, this explanation about this issue does not exist because the explanation about this issue changes now.